receiver/splunkhecreceiver/receiver.go

```diff
@@ -458,7 +460,12 @@ func (r *splunkReceiver) failRequest(
 }
 
 func (r *splunkReceiver) handleHealthReq(writer http.ResponseWriter, _ *http.Request) {
-	writer.WriteHeader(200)
+	writer.Header().Add("Content-Type", "application/json")
+	writer.WriteHeader(http.StatusOK)
+	_, err := writer.Write([]byte(responseHecHealthy))
+	if err != nil {
```

Comment:
Since we aren't doing anything with this error I think we can ignore its return from the `Write` call with a `_`.

Reply:
Yes, although my IDE shows "Unhandled error" warnings, I think It doesn't make sense as the value is always the same and hardcoded. I replace this line with bare `writer.Write([]byte(responseHecHealthy))`

Reply:
Ok, linter failed, you were right. Now it is `_, _ = writer.Write([]byte(responseHecHealthy))`